src/client/app.tsx

```diff
@@ -151,7 +141,7 @@ export default function Chat() {
             </div>
           )}
 
-          {agentMessages.map((m: Message, index) => {
+          {agentMessages.map((m: UIMessage, index) => {
             const isUser = m.role === "user";
             const showAvatar =
               index === 0 || agentMessages[index - 1]?.role !== m.role;
```

Comment:
seeing duplicate div key's giving some warnings in console below (likely unrelated this change), but suggestion would be:
```
return (
  <div key={`${m.id}-${index}`}>
 ...
 ```
```